src/codelabs/iterables.md

```diff
@@ -1,7 +1,22 @@
 ---
 title: Iterable collections
 description: An interactive guide to using Iterable objects such as lists and sets.
+js: [{url: 'https://dartpad.dev/inject_embed.dart.js', defer: true}]
 ---
+<?code-excerpt replace="/ *\/\/\s+ignore_for_file:[^\n]+\n//g; /(^|\n) *\/\/\s+ignore:[^\n]+\n/$1/g; /(\n[^\n]+) *\/\/\s+ignore:[^\n]+\n/$1\n/g"?>
```

Comment:
What is this doing?

Reply:
I have no idea. :(

@chalin can you recall?

Reply:
It strips out analyzer `ignore*` directives from all code samples, since we don't usually want such directives present in the excepts (unless we're actually documenting the ignore directives :)).

Reply:
Thanks, @chalin!

Reply:
Btw, you don't need to include this in every `*.md` file that has code samples, only those with code samples that contain ignore analyzer directives. (FYI :))